tools/esdlcmd/esdlcmd_monitor.cpp

```diff
@@ -564,7 +579,7 @@ class EsdlMonitorCmd : public EsdlConvertCmd
                 }
                 else
                 {
-                    xpath.setf("EsdlStruct[@name='%s']", partElement->queryProp("@complex_type"));
+                    VStringBuffer xpath("EsdlStruct[@name='%s']", partElement->queryProp("@complex_type"));
```

Comment:
What is the advantage of declaring a new xpath variable instead of reusing the existing?

Reply:
Duh. Trying to do too many things at once.